integration/integration/src/main/java/com/example/integration/IntegrationApplication.java

```diff
@@ -89,6 +89,7 @@ WireTap loggingWireTap() {
 	MessageHandler loggingHandler() {
 		LoggingHandler loggingHandler = new LoggingHandler(LoggingHandler.Level.TRACE);
 		loggingHandler.setLoggerName("tracing.data");
+		loggingHandler.setLogExpressionString("#jsonPath(payload.toString(), '$')");
```

Comment:
You've added this log expression but I don't see any assertions on it. Is this intended?

Reply:
The goal of this change is to be sure that native image is build OK and its run is successful.
Without respective change in Spring Integration it failed before.
To see if log expression works well is out of this project scope: we indeed test it on Spring Integration side.
I thought I explained it well in the PR header...

Reply:
Ah, i see. Startup fails if this wouldn't work. Thanks for the explanation!